databend-jdbc/src/main/java/com/databend/jdbc/DatabendDatabaseMetaData.java

```diff
@@ -889,6 +886,20 @@ public ResultSet getTables(String catalog, String schemaPattern, String tableNam
         buildFilters(sql, filters);
         sql.append("\nORDER BY table_type, table_catalog, table_schema, table_name");
 
+        if (types == null || types.length == 0 || Arrays.stream(types).allMatch(t -> t.equalsIgnoreCase("VIEW"))) {
```

Comment:
IMHO, using `table` as the default table type is better and has much better compatibility。